sympy/parsing/tests/test_latex.py

```diff
@@ -248,10 +250,7 @@ def test_parseable():
         assert parse_latex(latex_str) == sympy_expr
 
 # At time of migration from latex2sympy, should work but doesn't
-FAILING_PAIRS = [
-    (r"\log_2 x", _log(x, 2)),
-    (r"\log_a x", _log(x, a)),
-]
+FAILING_PAIRS = []
 
 def test_failing_parseable():
     from sympy.parsing.latex import parse_latex
```

Comment:
Maybe `test_failing_parseable` isn't needed any more

Reply:
Ok, I removed it.